open-metadata-distribution/open-metadata-packages/open-metadata-caller-package/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>open-metadata-packages</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+
+    <name>Open Metadata Caller Package</name>
+    <description>
+        Provides the basic package for calling egeria from a client.
```

Comment:
Since this is a package of our rest services (spring) it's what is needed in the server to support client access. This would be distinct from a client package which includes the java classes to make those rest calls easier, so I think I'd name it something like 'provides support for egeria clients in a server' or similar?